Week01/Problem01/aster_01.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jul 24 17:31:19 2017
+
+@author: Aster
+"""
+
+import math
+
+def ProjectEuler_problem1(n1,n2,N):
+    """
+    If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. 
```

Comment:
Check out the conventions for writing function docstrings [here](https://www.python.org/dev/peps/pep-0257/).  In the future, the sort of documentation you've given here would be better in the docstring for the *module*, i.e., in the file header, rather than within a particular function.

Also, you don't need to sandbox all your script's code within a single function; you only really need to do this if you want to re-use your code elsewhere.